### ISSUE TITLE
Note collision

### ISSUE THREAD

**Issue by bbloomf** [rism-digital/verovio#2580]
**Describe the problem**
A note in one layer is colliding with the a note at the same time and on the same pitch in a different layer on the same staff

**To Reproduce**
1. Load the data attached in https://editor.verovio.org/
2. The two F notes are on top of each other

**Expected behavior**
The F notes should appear horizontally adjacent to each other.  Interestingly, if the order of the layer elements is reversed, such that layer 2 (containing the whole note / semibreve) comes first in the XML, it renders correctly

**Input data**
[example MEIs.zip](https://github.com/rism-digital/verovio/files/7883733/example.MEIs.zip)
```
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
  <meiHead>
    <fileDesc>
      <titleStmt>
        <title></title>
      </titleStmt>
      <pubStmt></pubStmt>
    </fileDesc>
  </meiHead>
  <music>
    <body>
      <mdiv>
        <score>
          <scoreDef>
            <staffGrp>
              <staffDef n="1" lines="5" clef.shape="F" clef.line="4" key.sig="1f" meter.count="4" meter.unit="4" meter.sym="common" />
            </staffGrp>
          </scoreDef>
          <section>
            <measure>
              <staff n="1">
                <layer n="1">
                  <note xml:id="note-136" dur="2" oct="3" pname="f" />
                  <note xml:id="note-137" dur="2" oct="3" pname="a" />
                </layer>
                <layer n="2">
                  <note xml:id="note-138" dur="1" oct="3" pname="f" />
                </layer>
              </staff>
              <slur startid="#note-136" endid="#note-137" />
            </measure>
          </section>
        </score>
      </mdiv>
    </body>
  </music>
</mei>
```

**Verovio information**
 - Version of Verovio: 3.6.0, as well as version running on https://editor.verovio.org/

**Screenshots**
with layer 1 first in the xml file:
![image](https://user-images.githubusercontent.com/923595/149814416-5bcf34c3-0eb0-4f61-871c-a08031e98f17.png)

with layer 2 first in the xml file:
![image](https://user-images.githubusercontent.com/923595/149814459-b3327167-4b90-426e-abcc-8eee73edf6e3.png)

**Comment by DavidBauer1984** [comment]
Is it possible that the editor is running an ancient version of Verovio? I can only reproduce the problem there. I've tried version 3.6.0, 3.7.0 and the latest develop: all examples look good here.

Example _not-working_
![not-working](https://user-images.githubusercontent.com/63608463/150147510-2676e8b0-d891-4a3e-ac97-611bc9777dee.png)

Example _working_
![working](https://user-images.githubusercontent.com/63608463/150147575-1c7def8c-586a-484f-a49c-172bd987ca96.png)

What's interesting is that also the stem directions are different.

**Comment by lpugin** [comment]
> Is it possible that the editor is running an ancient version of Verovio?

It is running `3.9.0-dev-58431d0` which can see it in the browser console, so may be there a cache issue somewhere?

**Comment by bbloomf** [comment]
Sorry about this; it appears I included the wrong files in the zip.  Here are the correct ones:
[example MEIs.zip](https://github.com/rism-digital/verovio/files/7897982/example.MEIs.zip)

(The stem directions was a give away.)  I had also tested both cases with layers reversed such that the 1st layer had the single F, and the 2nd layer had the movement, and it worked in that case whichever order the layers were given in the XML as seen in @DavidBauer1984 s attachments.  The MEI code I included directly was correct though and does produce colliding notes both in the editor and in the command line.  Sorry for including the wrong MEI files in the .zip.
